package.json

```diff
@@ -12,13 +12,14 @@
     "flow": "flow",
     "gen-snapshot-tests": "node utils/gen-snapshot-tests.js && prettier --write packages/*/generated-snapshot.test.js",
     "jest": "SNAPSHOT_INLINE_APHRODITE=1 jest",
-    "test": "npm run lint && npm run gen-snapshot-tests && npm run flow && npm run jest",
+    "test": "npm run lint && npm run gen-snapshot-tests && npm run build:all && npm run flow && npm run jest",
```

Comment:
`yarn build:all` and `yarn flow`. Let's standardize on `yarn` and drop all these `npm` refs.